spec/support/helpers/database_helper.rb

```diff
@@ -37,7 +50,7 @@ def destroy_test_models
   def destroy_test_model(name)
     model_name = name.to_s.classify
     clazz = Object.const_get(model_name)
-    if !clazz.nil?
+    unless clazz.nil?
```

Comment:
Why are we skipping cleanup if the const wasn't set? If it's present in the cleanup list, we could do some cleanup, like dropping the tables we created before we threw.